products/error_tracking/frontend/components/ExceptionAttributesPreview/ReleasesPreview/releasePreviewLogic.ts

```diff
@@ -0,0 +1,69 @@
+import { kea, path } from 'kea'
+import { loaders } from 'kea-loaders/lib'
+
+import api from 'lib/api'
+import 'lib/components/Errors/stackFrameLogic'
+import { ErrorTrackingStackFrame, RawEventExceptionRelease } from 'lib/components/Errors/types'
+
+import type { releasePreviewLogicType } from './releasePreviewLogicType'
+
+export const releasePreviewLogic = kea<releasePreviewLogicType>([
+    path([
+        'products',
+        'error_tracking',
+        'frontend',
+        'components',
+        'ExceptionAttributesPreview',
+        'ReleasesPreview',
+        'releasePreviewLogic',
+    ]),
+
+    loaders(() => ({
+        release: [
+            undefined as RawEventExceptionRelease | undefined,
+            {
+                loadRelease: async (dto: {
```

Comment:
What does `dto` mean in this context?

Reply:
No longer applies. It now accepts one argument.

But by `dto` I just meant generic name for object I pass to the method. Like `props` or `params`. But I really wanted to avoid `props` as this is already a keyword in `kea`.